.travis.yml

```diff
@@ -48,14 +43,8 @@ matrix:
           compiler: gcc-5
           env: CONFIG_OPTS="no-asm --strict-warnings -fno-sanitize-recover -fsanitize=address -fsanitize=undefined enable-rc5 enable-md2"
         - os: linux
-          compiler: clang-3.6
-          env: CONFIG_OPTS="no-engine"
-        - os: linux
-          compiler: gcc
-          env: CONFIG_OPTS="no-engine"
-        - os: linux
-          compiler: gcc-5
-          env: CONFIG_OPTS="no-engine"
+          compiler: clang
+          env: CONFIG_OPTS="no-engine" BUILDONLY="yes"
```

Comment:
It's a build break test; I picked one at random. Doesn't really seem to matter which compiler you use but I can change it to the newer clang if y'all think that's better.